common/config/config_test.go

```diff
@@ -13,17 +14,31 @@ func TestToString(t *testing.T) {
 	require.NotEmpty(t, cfg.String())
 }
 
-func TestEmbeddedTemplateOnlyDiffersFromDockerByComment(t *testing.T) {
-	embeddedContent, err := os.ReadFile("config_template_embedded.yaml")
+func TestEmbeddedTemplateConsistentWithDockerTemplate(t *testing.T) {
```

Comment:
```suggestion
func TestEmbeddedTemplateConsistentWithDockerTemplate(t *testing.T) {
```
this test in no longer relevant since there's a syntax different between the 2 files. docker-builds still expects the `dockerize` syntax.